cmd/frontend/backend/user_emails.go

```diff
@@ -311,18 +311,28 @@ func (e *userEmails) SendUserEmailOnFieldUpdate(ctx context.Context, id int32, c
 }
 
 var updateAccountEmailTemplate = txemail.MustValidate(txtypes.Templates{
-	Subject: `Update to your Sourcegraph account ({{.Host}})`,
+	Subject: `({{.Change}}) on your Sourcegraph account ({{.Host}})`,
```

Comment:
Is having the change in parens intentional? I think this will render a subject like:

```
(An email was removed) on your Sourcegraph account (foobar.sourcegraph.com)
```

Might be clearer if we remove the parens:

```suggestion
	Subject: `{{.Change}} on your Sourcegraph account ({{.Host}})`,
```